controllers/userController.js

```diff
@@ -0,0 +1,45 @@
+const sharp = require("sharp");
+const User = require("../models/User");
+
+// Fetching all profiles and your profile
+module.exports.fetch_all_profiles_get = async (req, res) => {
```

Comment:
suggested to rename all these exported functions to camelCase

In addition, instead of exporting them individually, just put these functions in a variable such as `const fetchAllProfilesGet` and at the end of the file, you can export the ones as needed 

for example: https://stackoverflow.com/a/43875150/14204824